mosdef_dihedral_fit/dihedral_fit/fit_dihedral_with_gomc.py

```diff
@@ -597,6 +607,19 @@ def fit_dihedral_with_gomc(
     else:
         raise TypeError(print_error_value)
 
+    # Check if 'combining_rule' is correct
+    # Set the VDWGeometricSigma variable from the combining_rule
+    if combining_rule == None:
```

Comment:
Yeah this is a nice enough way to merge the APIs for dihedral-fit and mosdef-gomc. Just means the top level arguments are going to much more clear and easy to find, and it's clear from the input what the arguments for mosdef-gomc would correspond to.

Reply:
cool!